### PR TITLE
LPS-167865 Created alert for when only one subtask is selected

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Review by welitonpe — APPROVED** [review]

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 1]
```diff
@@ -93,6 +93,18 @@ const TestFlowTasks = () => {
...
+	const getAlert = () => {
+		if (selectedRows.length === 1) {
+			return [
+				{
+					text: i18n.translate(
+						'please-select-at-least-two-subtasks-to-merge'
+					),
+				},
+			];
+		}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:105]: As you said that more alerts may appear, add it conditionally this way: 🦅 

```suggestion
               const alerts = [];
               
		if (selectedRows.length === 1) {
			alerts.push({
				text: i18n.translate(
					'please-select-at-least-two-subtasks-to-merge'
				)}
		}
		
		return alerts;
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 2]
```diff
@@ -93,6 +93,18 @@ const TestFlowTasks = () => {
 		return <Loading />;
 	}
 
+	const getAlert = () => {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:96]: 😺 

```suggestion
	const getFloatingBoxAlerts = () => {
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 3]
```diff
@@ -327,6 +339,7 @@ const TestFlowTasks = () => {
 				>
 					{(_, dispatch) => (
 						<FloatingBox
+							alerts={getAlert()}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:342]: ⚡ 

```suggestion
							alerts={getFloatingBoxAlerts()}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Good job my friend, take a look 👍🏻

**Comment by rafaelfs98** [comment]
fixed @kevenleone

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**